functional-tests/features/forms/fill_select.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Forms
+    As a kakunin user
+    I want to check options
+
+    Scenario: Fill and check form fields
+        Given I visit the "main" page
+        When I click the "formSelectLink" element
+        Then the "simpleSelectForm" page is displayed
+        And there is "selectPerson" element with following dropdown list options:
```

Comment:
I'm not sure if `selectPerson` should point to select. It would be better if it's pointing to `option` as array

Reply:
I realised that i made a mistake in logic, it's already fixed. 
You mean we should ask for option selector like `this.selectXOptions  = $('somkind-of-select-css').$$('option');` ?